service/router.go

```diff
@@ -30,6 +30,7 @@ func setupRouter(h handler) *mux.Router {
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}", h.updateTarget).Methods(http.MethodPatch)
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}/operations", h.createWorkflowFromGit).Methods(http.MethodPost)
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}/workflows", h.listWorkflows).Methods(http.MethodGet)
+	r.HandleFunc("/projects/{projectName}/tokens", h.createToken).Methods(http.MethodPost)
```

Comment:
Did we decide to keep this as a POST instead of a GET?

Reply:
since this is a create of a new resource then it should be a POST

Reply:
currently we are accepting any json body in the POST request, aka not validating the input body. We can keep that as-is and just validate requests if/when that time comes in the future, or @bkono suggested that we may be interested in passing some kind of reference_id in the request body. This would allow for idempotency and bookkeeping of transactions e.g. replaying the request without generating a new token. Thoughts on this?

Reply:
If that's the case, I'd keep it as POST. Gives more flexibility in the future.